src/molecules/ShoppingCartV2/ShoppingCart.pcss

```
@@ -5,6 +5,11 @@
     }
   }
 
+  &__disclaimer {
+    font-size: 0.875rem;
+    padding: 0.5rem 0 0 1rem;
```

Ikke noe padding på høyre side, vil vel gå helt ut i kanten på små flater?
```suggestion
    padding: 0.5rem 1rem 0 1rem;
```